docs/developers/v2/fork-yearn-ui.md

```diff
@@ -0,0 +1,219 @@
+if (chainID === 1 || chainID === 1337) {
+  if (toAddress(vault.address) === toAddress('0xdA816459F1AB5631232FE5e97a05BBBb94970c95')) //DAI
+    vault.categories = ['All', 'Decentralized', 'Crypto Backed', 'Fiat Backed'];
+  if (toAddress(vault.address) === toAddress('0xa354F35829Ae975e850e23e9615b11Da1B3dC4DE')) //usdc
+    vault.categories = ['All', 'Centralized', 'Fiat Backed'];
+  if (toAddress(vault.address) === toAddress('0x7Da96a3891Add058AdA2E826306D812C638D87a7')) //usdt
+    vault.categories = ['All', 'Centralized', 'Fiat Backed'];
+  if (toAddress(vault.address) === toAddress('0xFD0877d9095789cAF24c98F7CCe092fa8E120775')) //yvTUSD
+    vault.categories = ['All', 'Centralized', 'Fiat Backed'];
+  if (toAddress(vault.address) === toAddress('0x378cb52b00F9D0921cb46dFc099CFf73b42419dC')) //yvLUSD
+    vault.categories = ['All', 'Decentralized', 'Crypto Backed'];
+} else if (chainID === 250) {
+  if (toAddress(vault.address) === toAddress('0xEF0210eB96c7EB36AF8ed1c20306462764935607')) //yvUSDC
+    vault.categories = ['All', 'Centralized', 'Fiat Backed'];
+  if (toAddress(vault.address) === toAddress('0x637eC617c86D24E421328e6CAEa1d92114892439')) //yvDAI
+    vault.categories = ['All', 'Decentralized', 'Crypto Backed', 'Fiat Backed'];
+  if (toAddress(vault.address) === toAddress('0x148c05caf1Bb09B5670f00D511718f733C54bC4c')) //yvUSDT
+    vault.categories = ['All', 'Centralized', 'Fiat Backed'];
+  if (toAddress(vault.address) === toAddress('0x0A0b23D9786963DE69CB2447dC125c49929419d8')) //yvMIM
+    vault.categories = ['All', 'Decentralized', 'Crypto Backed'];
+}
+_vaults.push(vault);
+```
+
+Here is what the new customized filtering should look like after applying all the above changes:
+
+![](https://i.imgur.com/cLfcNr4.png)
+
+## Change Logo
+
+Replace the logo image with either another image or a simple text:
+
+```jsx title="pagex/app.tsx"
+// pagex/app.tsx line 71
+
+<h1>Cozy Stables Finance</h1>
+```
+
+Changing the above line will make the page header look like this now:
+
+![](https://i.imgur.com/Lt0kFQM.png)
+
+## Change Colors
+
+The default color theme uses HSL, this allows for quicker customization of the entire theme with less variables, a quick primer on HSL:
+
+**HSL** stands for Hue, Saturation, Lightness and is represented on tailwind like this:
+
+```css
+--variable: 200 100% 50%;
+```
+
+which will be compiled to:
+
+```css
+--variable: hsl(200, 100%, 50%);
+```
+
+**Hue** is measured in degrees:
+- 0° is red
+- 120° is green
+- 240° is blue.
+
+![hue examples](https://i.imgur.com/ddaVLBc.png)
+
+**Saturation** is how colorful or vivid a color looks:
+- 0% saturation is grey (no color)
+- 100% saturation is vibrant and intense.
+
+![saturation examples](https://i.imgur.com/PkTorUr.png)
+
+**Lightness** measures how close a color is to black or white:
+- 0% lightness is pure black
+- 50% lightness is a pure color at the given hue.
+- 100% lightness is pure white
+
+![lightness examples](https://i.imgur.com/A8coxLo.png)
```

Comment:
Can we link people to a resource with about HSL instead of explaining it all? It seems to be out of the scope of the doc